ipi/inputs/forcefields.py

```diff
@@ -614,3 +616,147 @@ def fetch(self):
             dopbc=self.pbc.fetch(),
             threaded=self.threaded.fetch(),
         )
+
+
+class InputFFCommittee(InputForceField):
+    default_help = """A class to consolidate multiple FF inputs in a single XML field.
```

Comment:
It sounds completely cryptic to me this first phrase -- no idea what it means to consolidate FF inputs into a XML field. And the phrase implies one can use an XML field for uncertainty estimation (which I don't think was the exact idea). More helpful description, please?